etc/ssdlc_compliance_report.md

```diff
@@ -0,0 +1,40 @@
+# Mongo C++ Driver SSDLC Compliance Report
+
+## Release Creator
+
+- See [C/CXX Release Info](https://docs.google.com/spreadsheets/d/1yHfGmDnbA5-Qt8FX4tKWC5xk9AhzYZx1SKF4AD36ecY/edit?usp=sharing).
+
+## Process Document
+
+- See [go/how-we-develop-software-doc](http://go/how-we-develop-software-doc).
```

Comment:
Suggest leaving as a placeholder.

```suggestion
Unavailable. <!-- TODO: Replace with link to document referenced in CXX-3007 once available. -->
```

I expect the `go` link is inaccessible to external users. [MongoDB Software Security Development Lifecycle Policy](https://docs.google.com/document/d/1u0m4Kj2Ny30zU74KoEFCN4L6D_FbEYCaJ3CQdCYXTMc/edit#bookmark=id.ic9o2y62f82x) notes:
> For products with publicly available source code, our practice of how we develop software is expected to be available in a public facing document.

Reply:
Expectation was that the GoLink will be updated to the public-facing document, but forgot that GoLinks themselves are internal-only. 🤦 Will remove reference as suggested.